ZenPacks/zenoss/OpenStackInfrastructure/datasources/NovaServiceStatusDataSource.py

```diff
@@ -179,7 +179,7 @@ def onSuccess(self, result, config):
             'summary': 'Nova Status Collector: successful collection',
             'severity': ZenEventClasses.Clear,
             'eventKey': 'openStackNovaServiceCollectionError',
-            'eventClassKey': 'openstackRestored',
+            'eventClassKey': 'openStackFailure',
```

Comment:
On failure we create events with evenClassKey 'openStackFailurfe', therefore in success case to clear this failure event we need to create event with the same eventClassKey